tests/testthat/test-html-tweak.R

```diff
@@ -420,3 +420,60 @@ test_that("tweak_tabsets() with tab pills, fade and second tab active", {
   new_html <- tweak_tabsets(xml2::read_html(html))
   expect_snapshot_output(cat(as.character(new_html)))
 })
+
+
+# tweak_reference_topic_html ---------------------
+test_that("tweak_reference_topic_html() works", {
```

Comment:
Are these tests too cumbersome? :thinking:

Reply:
The snapshot depends on the Pandoc version so I'm not sure whether the test should be removed, or skipped when the Pandoc version is not something (something being the one installed on CI).

Reply:
An alternative being tests with XPath queries on the HTML output (e.g. we still expect some spans of class "fu" no matter the Pandoc version).